src/Module.php

```diff
@@ -126,12 +125,23 @@ class="edit-entry"
                 }
             }
         );
+
+        // Define template directory for site
+        if (($firstSegment = Craft::$app->request->segments[0] ?? null) === PartsKit::getConfig('directory')) {
```

Comment:
both here and in `Bundle.php`...why is this set to `$firstSegment`? considering you do this logic twice, how would you feel about a parts kit service method like `PartsKit::isRequest` or something?

Reply:
Ahh guess i don't need to do it like this now. But basically that means you can use `$firstSegment` inside of the condition.

But yeah, I like your idea of having a static method